src/js/components/Select/SelectContainer.js

```diff
@@ -376,6 +408,7 @@ const SelectContainer = forwardRef(
                   setActiveIndex(-1);
                   onSearch(nextSearch);
                 }}
+                onFocus={() => setActiveIndex(-1)}
```

Comment:
This was something I inadvertently missed in my original changeset.

Without it, after cycling focus back to the search input using the keyboard, the down key won't navigate focus down to the first option in the list as one might expect.